src/resolver/tsconfig_json.zig

```diff
@@ -33,6 +33,7 @@ pub const TSConfigJSON = struct {
     // More info: https://github.com/microsoft/TypeScript/issues/31869
     base_url_for_paths: string = "",
 
+    extends: ?string = null,
```

Comment:
To slightly reduce struct size (good for perf), we can make `string` an empty string by default instead of a nullable string
```suggestion
    extends: string = "",
```